src/UglyToad.PdfPig/Content/Letter.cs

```diff
@@ -65,5 +82,26 @@ public override string ToString()
         {
             return $"{Value} {Location} {FontName} {PointSize}";
         }
+
+        private TextDirection getTextDirection()
```

Comment:
Just a minor style thing, this should be `GetTextDirection` even for private methods.